engine/src/main/java/org/hibernate/search/engine/backend/spi/BackendBuildContext.java

```diff
@@ -22,6 +23,8 @@
 
 	ResourceResolver resourceResolver();
 
+	ServiceResolver serviceResolver();
```

Comment:
If you follow my recommendation from other comments, you can remove this.

Reply:
I'll remove it.